client/data/conferences.json

```diff
@@ -11,6 +11,12 @@
       "start_date": "2019-06-01",
       "end_date": "2019-06-02",
       "locations": ["Arena Berlin. Eichenstraße 4, 12435 Berlin, Germany"]
+    },
+    "2": {
```

Comment:
To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.